funnel/static/translations/hi_IN/messages.json

```diff
@@ -1473,14 +1467,19 @@
         "लिखें, दूसरे से जुड़ें और प्रतिक्रिया पाएं."
       ],
       "Unscheduled projects": [null, "अनिर्धारित प्रोजेक्ट"],
-      "Back to the project": [null, "प्रोजेक्ट पर वापस जाएं"],
-      "Edit details": [null, "विवरण संपादित करें"],
+      "This project will not be listed as it has no sessions in the schedule": [
+        null,
+        ""
+      ],
+      "This project is currently not published. Visit <a href=\"%(url)s\">settings</a> to publish": [
```

Comment:
Settings link should not be in the text. It should have an arrow outside the translation, like we use in notifications (where the whole text is a link). Also remove the word "currently" as it's implied when the banner is shown.

![CTA in updates feed](https://user-images.githubusercontent.com/26695/107610059-affd2900-6c66-11eb-8111-4cb176beb4f4.png)